cfme/tests/services/test_provision_stack.py

```diff
@@ -26,73 +26,127 @@
 AWS_TEMPLATE = """
```

Comment:
I completely agree.  Especially since I need to start expanding the number of json templates for azure as well.  Added JIRA card RHCFQE-1750 and assigned to myself.

Do I just create a directory inside this directory?  I'll find you when I start on the task and we'll find a nice place to organize these.